tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml

```diff
@@ -0,0 +1,20 @@
+---
+addr: '0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874'
+decimals: 18
+description: >- Tokenized 1 Troy Ounce Gold Bar, Redeemable in Precious Metal (Gold)
```

Comment:
Description should probably include a bit more info. Eg., see the description in the original request: https://github.com/forkdelta/tokenbase/issues/1568#issue-349772441

On the technical side, text after `>-` should be placed on the next line, indented with 2 spaces, like so:
```yaml
description: >-
  VGLZ1 is a sub-token of VERI (VERITASEUM) Ecosystem.
  
  VGLZ1 is a digital token asset representing 1oz physical fine .999 gold that can be exchanged
  and then redeemed via the Veritaseum website, or VeADIR Dapp after KYC is provided.
```
Single line breaks will be converted to a space, two line breaks (or two spaces followed by a line break) will produce a new line (like Reddit comments). (see: https://github.com/forkdelta/tokenbase/blob/master/README.md#description)